### PR TITLE
Update HUMAnN and add the new tools

### PR DESCRIPTION
**Pull request by bebatut**
Hi,

I updated HUMAnN2 to the latest version and add the new tools they developed.
I also tried to fix some styling issues

Bérénice

### CODE REVIEW

File: tools/humann2/humann2_barplot.xml [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        --output 'output.png'
+    #elif $format == 'pdf'
+        --output 'output.pdf'
+    #elif $format == 'svg'
+        --output 'output.svg'
+    #end if
+    ]]></command>
+    <inputs>
+        <param argument="--input" type="data" format="tsv,tabular" label="HUMAnN2 table with optional metadata"/>
+        <param name="focal_feature" argument="--focal-feature" type="text" value="" label="Feature ID of interest (give ID not full name)"/>
```
Comment (bebatut) [inline tools/humann2/humann2_barplot.xml]: Done

File: tools/humann2/humann2_barplot.xml [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+            <param name="remove_zeroes" value=""/>
+            <param name="width" value="5"/>
+            <param name="height" value="8"/>
+            <param name="width" value="4"/>
+            <param name="format" value="png"/>
+            <output name="barplot_png" file="barplot4.png" ftype="png" compare="sim_size"/>
+        </test>
+    </tests>
+    <help><![CDATA[
+@HELP_HEADER@
```
Comment (bebatut) [inline tools/humann2/humann2_barplot.xml]: Done

File: tools/humann2/humann2_split_stratified_table.xml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+        <import>humann2_macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <expand macro="version"/>
+    <command detect_errors="exit_code"><![CDATA[
+humann2_split_stratified_table
+    --input '$input'
+    --output 'output'
+    && 
+    ls output
```
Comment (bebatut) [inline tools/humann2/humann2_split_stratified_table.xml]: I removed it (left over from debugging)

File: tools/humann2/humann2_strain_profiler.xml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+humann2_strain_profiler first looks for (species, sample) pairs where (i) a large number of gene families within the species were identified (default: 500) and (ii) the mean abundance of detected genes was high (default: mean > 10 RPK). For species that meet these criteria, we can infer that absent gene families are likely to be truly absent, as opposed to undersampled. Simulations suggest that the cutoff of 10 RPK results in a false negative rate below 0.001 (i.e. for every 1000 genes identified as absent, at most one would be present but missed due to undersampling). For a given species, if at least two samples pass these criteria, the species and passing samples are sliced from the merged table and saved as a strain profile.
+Strain profiles can be additionally restricted to a subset of species (e.g. those from a particular genus) or to gene families with a high level of variability in the population (e.g. present in fewer than 80% of samples but more than 20% of samples). Additional thresholds (e.g. the minimum non-zero mean) can be configured with command line parameters.
```
Comment (bebatut) [inline tools/humann2/humann2_strain_profiler.xml]: Fixed

**Comment by bebatut** [comment]
Thanks @shiltemann

File: tools/humann2/humann2_barplot.xml (hunk 2 above)
Comment (bgruening) [inline tools/humann2/humann2_barplot.xml]: Images should be stored locally ...

File: tools/humann2/humann2_regroup_table.xml [hunk 5]
```diff
@@ -40,23 +39,23 @@
                     </param>
                     <when value="uniref50">
                         <param name="groups" type="select" label="Grouping options" help="(--groups)">
-                            <option value="uniref50_go">UniRef50 gene families into GO</option>
+                            <!--<option value="uniref50_go">UniRef50 gene families into GO</option>
```
Comment (bgruening) [inline tools/humann2/humann2_regroup_table.xml]: Should this be removed?

File: tools/humann2/humann2_regroup_table.xml [hunk 6]
```diff
@@ -75,32 +74,35 @@
         <data format="tsv" name="output_table" label="${tool.name} on ${on_string}: Regrouped table" />
     </outputs>
     <tests>
-        <test>
+        <!--<test>
```
Comment (bgruening) [inline tools/humann2/humann2_regroup_table.xml]: Still valid?

File: tools/humann2/humann2_regroup_table.xml (hunk 5 above)
Comment (bebatut) [inline tools/humann2/humann2_regroup_table.xml]: I am not sure. It was in the last version of humann2 and was useful. I do not know why they removed it in this version. I would like to ask them

File: tools/humann2/humann2_regroup_table.xml (hunk 6 above)
Comment (bebatut) [inline tools/humann2/humann2_regroup_table.xml]: Same as the last version

File: tools/humann2/humann2_barplot.xml (hunk 2 above)
Comment (bebatut) [inline tools/humann2/humann2_barplot.xml]: Done

**Comment by bgruening** [comment]
Thanks B!

**Pull request merged**